src/components/Dialog/CollectionsDialog.js

```diff
@@ -84,6 +106,8 @@ const CollectionsDialog = ({
 }) => {
   const [inputText, setInputText] = useState('');
   const [errorText, setErrorText] = useState('');
+  const navigation = useNavigation();
+  const route = useRoute();
```

Comment:
Should need to use `navigation` and `route` here.

Reply:
removed